pvlib/test/test_scaling.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pandas as pd
+
+import pytest
+from numpy.testing import assert_almost_equal
+
+from pvlib import scaling
+from conftest import requires_scipy
+
+
```

Comment:
anything that's mutable and reused in multiple tests should be put in a [`pytest.fixture`](https://docs.pytest.org/en/latest/fixture.html). Comments below try to address this for a few items and I'm hoping you can follow the pattern for the rest.

Reply:
Because of the mutable comment I left the ints outside fixtures. If that's not right, please let me know. otherwise I think I've got it.